lib/perl/Genome/Model/Build.pm

```diff
@@ -1395,6 +1395,11 @@ sub _launch {
     local $ENV{UR_DUMP_STATUS_MESSAGES} = 1;
     local $ENV{UR_COMMAND_DUMP_STATUS_MESSAGES} = 1;
 
+    # we don't use 'local' because we want this to persist through
+    # spawned workflows
+    $ENV{Genome::Config::spec('lsf_project_name')->env} =
+        sprintf("build/%s", $self->id);
```

Comment:
I don't understand the comment about not using `local`.  How will spawned workflows that jobs for this build produces know whether or not the environment variable was reset when this `_launch` completes?

It seems like this may cause the ID of nested builds to improperly be associated with jobs spawned by the parent build.